hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/SimpleRpcServer.java

```diff
@@ -69,6 +69,7 @@
  * put itself on new queue for Responder to pull from and return result to client.
  * @see BlockingRpcClient
  */
+@Deprecated()
```

Comment:
Shall we add a comment stating that this will be likely removed from HBase 3.0 onwards?

Reply:
Sure, will do that on commit.